pkg/rtc/participant.go

```diff
@@ -830,6 +720,64 @@ func (p *ParticipantImpl) RemoveSubscribedTrack(subTrack types.SubscribedTrack)
 	}
 }
 
+func (p *ParticipantImpl) UpdateSubscriptionPermissions(
+	permissions *livekit.UpdateSubscriptionPermissions,
+	resolver func(participantSid string) types.Participant,
+) error {
+	return p.uptrackManager.UpdateSubscriptionPermissions(permissions, resolver)
+}
+
+func (p *ParticipantImpl) SubscriptionPermissionUpdate(publisherSid string, trackSid string, allowed bool) {
```

Comment:
New method in `Participant` interface to send update about subscription permission.